c2rust-analyze/src/context.rs

```diff
@@ -293,6 +293,8 @@ pub struct GlobalAnalysisCtxt<'tcx> {
 
     pub fn_sigs: HashMap<DefId, LFnSig<'tcx>>,
 
+    pub known_fns: HashMap<Symbol, &'static KnownFn>,
```

Comment:
Please add a doc comment explaining what the key means (I believe it's the C/ASM symbol name of the libc function in question)

Reply:
Yeah, that's what it is.  I'll add docs.

Reply:
Documented in 17d40e86e2b88a35b29f1698272fbb1d57c31e0a.  It's also now changed to a `&'static str` vs. a `Symbol` in b608247330c051b7b7b86b9f40071080e1ed7d69 (see https://github.com/immunant/c2rust/pull/980#discussion_r1257173224).